frontend/src/features/books/BookList.jsx

```diff
@@ -52,7 +52,8 @@ return (
               <td style={{ padding: '10px' }}>{book.title}</td>
               <td style={{ padding: '10px' }}>{book.author}</td>
               <td style={{ padding: '10px' }}>{book.description}</td>
-              <td><Link to={`/books/${book.id}`}>View</Link></td>
+              <td><Link to={`/books/${book.id}`}>View  |</Link></td>
```

Comment:
@nshelokar-vf, is there any specific reason for using `|`?
If needed, can we use CSS?

Reply:
Yes. Now I am using CSS.